packages/mui-material/src/ListItemText/ListItemText.spec.tsx

```diff
@@ -13,7 +13,7 @@ function typographyPropsTest() {
     primaryTypographyProps={{
       component: 'a',
       // @ts-expect-error
-      htmlFor: 'not-existance-attribute',
+      htmlFor: 'non-existant-attribute',
```

Comment:
Isn't it "non-existent" (or even "nonexistent")?

Reply:
Oops, you're right haha. Fixed a typo with another one lol